oap-server/server-storage-plugin/storage-jdbc-hikaricp-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/mysql/MySQLTableInstaller.java

```diff
@@ -74,22 +74,22 @@ protected void createTableIndexes(JDBCHikariCPClient client,
             if (!modelColumn.isStorageOnly()) {
                 SQLBuilder tableIndexSQL = new SQLBuilder("CREATE INDEX ");
                 tableIndexSQL.append(model.getName().toUpperCase())
-                             .append("_")
-                             .append(String.valueOf(indexSeq++))
-                             .append("_IDX ");
+                        .append("_")
+                        .append(String.valueOf(indexSeq++))
+                        .append("_IDX ");
                 tableIndexSQL.append("ON ").append(model.getName()).append("(")
-                             .append(modelColumn.getColumnName().getStorageName())
-                             .append(")");
+                        .append(modelColumn.getColumnName().getStorageName())
+                        .append(")");
```

Comment:
Just to name a few, these code styles are obviously not what we set in the `codeStyle.xml`, I believe one of you may loss the code style settings in the IDE

Reply:
I don't know why, but the CI doesn't report in both ways.

Reply:
> I don't know why, but the CI doesn't report in both ways.

CI never reports this kind of styles, we have many existed codes that don't follow the styles yet, just wanted to remind that the settings may be lost in your local machine,